pkg/syfs/syfs.go

```diff
@@ -77,6 +78,14 @@ func configDir(dir string) string {
 		}
 	}
 
+	if !fs.IsReadable(homedir) {
+		homedir = "/"
```

Comment:
Please insert a debug message here saying the home directory isn't readable, changing home directory to /.